apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts

```diff
@@ -206,15 +207,54 @@ export async function calculatePoolStats({
   };
 }
 
-function calculateRoundAPR(
-  round: Round,
+function getWeeksApart(startAt: Date, endAt: Date) {
+  const millisecondsInAWeek = 7 * 24 * 60 * 60 * 1000;
+  const differenceInMilliseconds = Math.abs(
+    startAt.getTime() - endAt.getTime(),
+  );
+  return Math.floor(differenceInMilliseconds / millisecondsInAWeek);
+}
+
+function calculateAPRForDateRange(
```

Comment:
this function is a bit messed up right? It's getting APR inputs from elsewhere and calculating the veBAL APR within it, we should refactor this

Reply:
https://linear.app/bleu-llc/issue/BAL-877/refactor-apr-to-calculate-other-apr-inside-of-it